### ISSUE TITLE
Incorrect/missing 3D Secure data

### ISSUE THREAD

**Issue by abramenal** [processout/processout-node#21]
### TL;DR

[ThreeDS](https://github.com/processout/processout-node/blob/master/src/threeds.ts) object returns incorrect `status` and `version` data, and does not provide getters for multiple other fields.

### Details
According to the API documentation, 3DS data is provided in given format:
```json
"three_d_s": {
    "ares_trans_status": "Y",
    "challenged": true,
    "cres_trans_status": "N",
    "ds_trans_id": "",
    "fingerprint_completion_indicator": "",
    "fingerprinted": true,
    "server_trans_id": "",
    "status": "success",
    "version": "2"
}
```

This shape is then consumed by `ThreeDS` entity [here](https://github.com/processout/processout-node/blob/master/src/threeds.ts#L143).

There are 2 things that are incorrect:

1. SDK expects `Status` and `Version` fields, however in API response above those fields are called `status` and `version` accordingly.
2. Some of the fields are just missing in SDK while being available in API response:
  -  `ares_trans_status`
  - `cres_trans_status`
  - `ds_trans_id`
  - `fingerprint_completion_indicator`
  - `server_trans_id`

### Environment

SDK version: `8.2.8`

**Comment by abramenal** [comment]
Please let me know if you have any contribution guidelines as this thing is confusing:  `// The content of this file was automatically generated`
Not sure where exactly the actual source code is placed
